source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRIBaseCardElementRenderer.h

```diff
@@ -18,7 +18,8 @@ using namespace AdaptiveCards;
 
 + (CardElementType)elemType;
 
-- (UIView* )render:(UIView* )viewGroup
+- (UIView *)render:(UIView *)viewGroup
+            inputs:(NSMutableArray *)inputs
```

Comment:
>NSMutableArray [](start = 20, length = 14)

This should be NSArray, unless there's a reason the method should modify the array.